plinth/modules/dynamicdns/templates/dynamicdns_configure.html

```diff
@@ -0,0 +1,165 @@
+{% extends "base.html" %}
+{% comment %}
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses/.
+#
+{% endcomment %}
+
+{% load bootstrap %}
+
+{% block content %}
+
+  <form class="form" method="post">
+    {% csrf_token %}
+    <div id='dynamicdns-no-js'>
+      You have disabled Javascript. Dynamic form mode is disabled and 
+      some helper functions may not work <br>
+      (but the main functionality should work)
+      <br><hr>
+    </div>
+
+    {{ form|bootstrap }}
+
+    <input type="submit" class="btn btn-primary" value="Update setup"/>
+
+  </form>
+
+{% endblock %}
+
+{% block page_js %}
+  <script type="text/javascript">
+
+    (function($) {
```

Comment:
A general note:
It's good that you use a self-executing anonymous functions to not clutter the global namespace.
The functions in here (select_service etc.) are still global. You needed that earlier because some javascript function calls were in the html `attr`s, but it's not necessary anymore.